docs/release-notes/NuGet-6.5.md

```diff
@@ -0,0 +1,125 @@
+* Breaking change in .NET 8 - Environment.GetFolderPath returns incorrect path on Unix - [#12127](https://github.com/NuGet/Home/issues/12127)
+
+* [Bug]: nuget.exe strings from NuGet.Commands are not localized - [#12097](https://github.com/NuGet/Home/issues/12097)
+
+* [Bug]: Process argument string is too long when publishing in Visual Studio with static graph enabled - [#11968](https://github.com/NuGet/Home/issues/11968)
+
+* Provide solution for NuGet Error NU1012 - the pack error does not call out the problem files - [#11905](https://github.com/NuGet/Home/issues/11905)
+
+* [Bug]: Some CLI commands don't respect DOTNET_CLI_UI_LANGUAGE - [#11326](https://github.com/NuGet/Home/issues/11326)
+
+* Reduce UI thread switching when determining solution folder and if solution is open - [#11090](https://github.com/NuGet/Home/issues/11090)
+
+* Issue saving settings in Visual Studio - adding a source throws an exception - [#8407](https://github.com/NuGet/Home/issues/8407)
+
+* PackageDependencyGroup does not implement Equals correctly - [#6478](https://github.com/NuGet/Home/issues/6478)
+
+[List of commits in this release](https://github.com/NuGet/NuGet.Client/compare/6.5.0.154...6.4.1.21)
+
+### Community contributions
+
+Thank you to all the contributors who helped make this NuGet release awesome!
+
+* [Forgind](https://github.com/Forgind)
+  * [4970](https://github.com/NuGet/NuGet.Client/pull/4970) Have SDK resolver always log an error when SDK resolution in unsuccessful
+* [marcin-krystianc](https://github.com/marcin-krystianc)
+  * [4952](https://github.com/NuGet/NuGet.Client/pull/4952) PrivateAssets for central transitive dependencies should flow regardless whether the parent node is a project or a package
+* [marcin-krystianc](https://github.com/marcin-krystianc)
+  * [4950](https://github.com/NuGet/NuGet.Client/pull/4950) Fix include flags calculation for transitively pinned centrally managed dependencies
+* [drewnoakes](https://github.com/drewnoakes)
+  * [4891](https://github.com/NuGet/NuGet.Client/pull/4891) Display documents from packages in the dependencies tree
+* [kvpt](https://github.com/kvpt)
+  * [4790](https://github.com/NuGet/NuGet.Client/pull/4790) Add support for restoring slnf file from command line
+* [AtariDreams](https://github.com/AtariDreams)
+  * [4863](https://github.com/NuGet/NuGet.Client/pull/4863) Simplification of Linq Any method
+* [davidegiacometti](https://github.com/davidegiacometti)
+  * [4840](https://github.com/NuGet/NuGet.Client/pull/4840) Refactor PackageDependencyGroup Equals and GetHashCode
+* [danjagnow](https://github.com/danjagnow)
+  * [4843](https://github.com/NuGet/NuGet.Client/pull/4843) Updated NU1012 error message to display item paths
+**TODO: Issues that could not be categorized. Make sure the issue has the correct milestone (if required) or an appropriate Type label - StillOpens:**
+
+* Add integration tests dotnet list package json outputs - [#12167](https://github.com/NuGet/Home/issues/12167)
```

Comment:
Issues about tests shouldn't be in the release notes, since they don't impact customers.  I believe this one is incorrectly here because @erdembayar added two Type labels to the issue (Bug and Test)